TheMET/TheMET/ViewControllers/CatalogViewController.swift

```diff
@@ -12,23 +12,7 @@ import MetAPI
 
 class CatalogViewController: UIViewController {
     
-    private let metAPI = MetAPI()
-    
-    private let aboutButton: UIButton = UIButton()
-    
-    private let loadingCatalogView = LoadingPlaceholderView.construstView(configuration: .catalogLoading)
-    private let failedCatalogView = FailedPlaceholderView.constructView(configuration: .catalogFailed)
-    private let loadedCatalogView = CatalogContentView.constructView()
-    
-    private var loadedDepartments: [Department] = []
-    
-    var contentStatus:LoadingStatus<[CatalogCellData]> = .loading {
-        didSet {
-            self.updateContent()
-        }
-    }
-    
-    var loadingDepartmentIds: [Int] = []
+    private let departmentsSectionViewController: DepartmentsSectionViewController = DepartmentsSectionViewController()
```

Comment:
According to designs, beside the Section with departments there is also a new title "Departments"